store/sql_post_store.go

```diff
@@ -471,6 +473,8 @@ func (s SqlPostStore) Search(teamId string, userId string, terms string, isHasht
 				    ORDER BY CreateAt DESC
 				LIMIT 100`, searchType)
 
+			terms = strings.TrimRight(terms, specialSearchCharCutset)
```

Comment:
Actually what would happen with "(hello)", wouldn't it become "(hello" ?  I guess I should go read what TrimRight does.